addons/website/static/src/snippets/s_tabs/003.scss

```diff
@@ -0,0 +1,97 @@
+// Tabs
+.s_tabs_common {
+
+    --tabs-bg-color: #{$gray-100};
+    --tabs-link-color: #{$body-color};
+
+    .nav-underline {
+        --nav-underline-link-active-color:currentColor;
+    }
+
+    .s_tabs_content {
+        &.s_tabs_slide_up, &.s_tabs_slide_down, &.s_tabs_slide_left, &.s_tabs_slide_right {
+            > .tab-pane.fade {
+                transition: all 0.2s;
+            }
+            > .tab-pane.fade.show {
+                transform: translateX(0rem) translateY(0rem);
+            }
+        }
+        &.s_tabs_slide_up > .tab-pane.fade {
+            transform: translateY(-1rem);
+        }
+        &.s_tabs_slide_down > .tab-pane.fade {
+            transform: translateY(1rem);
+        }
+        &.s_tabs_slide_left > .tab-pane.fade {
+            transform: translateX(-1rem);
+        }
+        &.s_tabs_slide_right > .tab-pane.fade {
+            transform: translateX(1rem);
+        }
+    }
+
+    .s_tabs_main {
+        --card-cap-bg: initial;
```

Comment:
why 🤔

Reply:
Had a look and it looks safe to remove ([this line](https://github.com/odoo/odoo/commit/80fde992370e4474c27a383a61814ce1159f550c#diff-ecb18a1c6c2b57ad28b91104a9d9a392b10605fec61b3a9aa2c219c4c6eb4bf9R37) doing the job)